src/views/Export/ExportHeader/index.js

```diff
@@ -144,6 +186,16 @@ export default class ExportHeader extends React.PureComponent {
             || undefined
         );
 
+        // NOTE: structure and level depict the same thing but are different in structure
+        // levels require the sublevels to be named sublevels whereas struture requires
+        // sublevels to be names levels
+        // This cannot be fixed immediately in server as it requires migration
+        const report_levels = createReportStructureLevelForExport(reportStructure || emptyList)
```

Comment:
Let us handle emptyList inside the method itself
It will be a better base condition IMO